h2o-r/h2o-package/R/zzz.R

```diff
@@ -48,6 +48,13 @@
   registerS3method("print", "H2OExplanation", "print.H2OExplanation")
   .s3_register("repr", "repr_text", "H2OExplanation")
   .s3_register("repr", "repr_html", "H2OExplanation")
+  
+  # CoxPH
+  .s3_register("survival", "survfit", "H2OCoxPHModel")
+  
+  # H2OFrame
+  .s3_register("stats", "median", "H2OFrame")
```

Comment:
`stats` package is not imported in the package namespace so it has to be specified here.

Confusingly, `min`, `mean`, `max` are defined in `base` package that is imported so they don't have to be listed here.